core/templates/dev/head/pages/exploration_player/TutorCardDirective.js

```diff
@@ -83,12 +85,18 @@ oppia.directive('tutorCard', [
 
             $scope.hintsExist = Boolean(oppiaPlayerService.getInteraction(
               $scope.activeCard.stateName).hints.length);
+
+            $scope.contentAudioTranslations =
```

Comment:
Actually, it might have not been anything. I can't quite remember...

I do remember fussing over why the audio controls wasn't showing only to realize that I didn't have any audio translations on the card I was testing and that solved the problem. Maybe that was my "small fix" lol